pretrained.md

```diff
@@ -8,3 +8,6 @@ The pretrained embeddings are trained using
 | [English](http://www.sfs.uni-tuebingen.de/a3-public-data/finalfusion/english-skipgram-mincount-50-ctx-10-ns-5-dims-300.fifu) | skipgram            | [CoNLL 2017 raw](https://lindat.mff.cuni.cz/repository/xmlui/handle/11234/1-1989) | [Metadata](http://www.sfs.uni-tuebingen.de/a3-public-data/finalfusion/english-skipgram-mincount-50-ctx-10-ns-5-dims-300.txt)  |
 | [German](http://www.sfs.uni-tuebingen.de/a3-public-data/finalfusion/german-skipgram-mincount-30-ctx-10-dims-300.fifu)        | skipgram            | [TüBa-D/DP](https://github.com/sfb833-a3/tueba-ddp)                                                                         | [Metadata](http://www.sfs.uni-tuebingen.de/a3-public-data/finalfusion/german-skipgram-mincount-30-ctx-10-dims-300.txt)        |
 | [German](http://www.sfs.uni-tuebingen.de/a3-public-data/finalfusion/german-structgram-mincount-30-ctx-10-dims-300.fifu)      | structured skipgram | [TüBa-D/DP](https://github.com/sfb833-a3/tueba-ddp)                                                                         | [Metadata](http://www.sfs.uni-tuebingen.de/a3-public-data/finalfusion/german-structgram-mincount-30-ctx-10-dims-300.txt)      |
+| [German](http://www.sfs.uni-tuebingen.de/a3-public-data/finalfusion/german-structgram-mincount-30-ctx-10-dims-300-zipf-0.75-NEW.fifu)      | structured skipgram (new)| [TüBa-D/DP](https://github.com/sfb833-a3/tueba-ddp)                                                                         | [Metadata](http://www.sfs.uni-tuebingen.de/a3-public-data/finalfusion/german-structgram-mincount-30-ctx-10-dims-300-zipf-0.75-NEW.txt)       |
```

Comment:
We probably need better naming than `NEW`? AFAIR, we wanted to switch to a date-based format. We could keep the old embeddings on the server, but do we still want to link to them?

Reply:
File names / URLs would become pretty long, not sure how we should name them in that case... I'll try some formats

Reply:
Pushed out with dates appended, also changed the file names on turing accordingly. I removed the old structgram file from the listing but didn't change it on the server.